### DISCUSSION TITLE
Contribution

### DISCUSSION THREAD

**Discussion by luluna02** [pear-devs/pear-desktop#1689] [Q&A]
Hello, I find this project very cool!! I fixed one of the issues and would love to do more so ..I would like to know how I can become a contributor? I appreciate your replies :).

**Comment by ArjixWasTaken** [comment]
On GitHub, a contributor is someone who contributes code, sadly we can't change that.
But if you do have an impact via reporting bugs, translating text (or QOL enhancements like #1687), we could add you to the README.

On that note, maybe we should have a section for non-code contributors on the README

CC: @th-ch, @JellyBrick, @Su-Yong

**Reply by luluna02** [reply]
@ArjixWasTaken I want to contribute code, I already fixed the issue #1617 on my local env and I would like to work on #1687.

**Reply by ArjixWasTaken** [reply]
That's great!

[Here](https://github.com/firstcontributions/first-contributions/blob/main/gui-tool-tutorials/github-windows-vs-code-tutorial.md) is a guide on how you can create a fork, then create a branch for your changes and then a Pull Request to submit them to us.

If you are comfortable with the command line, I'd suggest you follow the [cli-tutorial](https://github.com/firstcontributions/first-contributions) instead!
